### PR TITLE
Improve message when an automation step was not executed

### PR DESCRIPTION
**Pull request by emontnemery**
## Proposed change

Improve message when an automation step was not executed

## Type of change

- [ ] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (thank you!)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Example configuration

```yaml

```

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue or discussion:
- Link to documentation pull request:

## Checklist

- [ ] The code change is tested and works locally.
- [ ] There is no commented out code in this PR.
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

[docs-repository]: https://github.com/home-assistant/home-assistant.io

### CODE REVIEW

File: src/translations/en.json [hunk 1]
```diff
@@ -2988,15 +2988,15 @@
               "blueprint_config": "Blueprint Config"
             },
             "path": {
-              "choose": "Select a node on the left for more information.",
+              "choose": "Select a step on the left for more information.",
               "default_action_executed": "The default action was executed because no options matched.",
-              "no_further_execution": "This node was not executed and so no further trace information is available.",
-              "disabled_node": "This node was disabled and skipped during execution so no further trace information is available.",
+              "no_further_execution": "This step was not executed and so no further trace information is available.",
+              "disabled_step": "This step was disabled and skipped during execution so no further trace information is available.",
```
Comment (bramkragten) [inline src/translations/en.json:2994]: Can we keep the same keys, so the translations are not lost?

Comment (emontnemery) [reply src/translations/en.json:2994]: Yes, but translations were added only 2 weeks ago and are not in a release yet, do we need to keep the keys? 🤔

Comment (bramkragten) [reply src/translations/en.json:2994]: They are probably already translates though, let me check

**Review by bramkragten — APPROVED** [review]

File: src/translations/en.json (hunk 1 above)
Comment (silamon) [reply src/translations/en.json:2994]: When you change the key of a translation, the old and new key of the translation are kept in localise. So, if it is already translated, the translation has to be redone. If it isn't, the translation needs to be translated twice. Next time, keep the key, then the translation is kept for history and doesn't need to be translated once or twice.

**Pull request merged**